packages/pytest-simcore/src/pytest_simcore/helpers/comparing.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
+import hashlib
+from concurrent.futures import ProcessPoolExecutor
+from pathlib import Path
+from typing import TypeAlias
+
+import aiofiles
+from servicelib.file_utils import create_sha256_checksum
+
+_FilesInfo: TypeAlias = dict[str, Path]
+
+
+def get_relative_to(folder: Path, file: Path) -> str:
+    return f"{file.relative_to(folder)}"
```

Comment:
what is the point of such a function that directly calls another?

Reply:
Yes.
I think it has no impact on the execution since this is praised with functions that access disk and/or network. It's simply more convenient.